schema.graphql

```diff
@@ -993,6 +993,8 @@ input UpdateOrganizationInput {
 }
 
 input UpdateUserInput {
+  id: ID
```

Comment:
@Shubh152 are you using this id anywhere ?

Reply:
No, removed it now